doc/source/conf.py

```diff
@@ -316,6 +316,9 @@
 plot_pre_code = """
 import warnings
 for key in (
+        'lsim2 is deprecated',  # Deprecation of scipy.signal.lsim2
+        'impulse2 is deprecated',  # Deprecation of scipy.signal.impulse2
+        'step2 is deprecated',  # Deprecation of scipy.signal.step2
```

Comment:
The only global changes are here, though I don't see how this could break something, as we're _adding_ more warnings to filter.